source/app/service-providers/fsal/index.ts

```diff
@@ -171,7 +171,7 @@ export default class FSAL extends ProviderContract {
     }
 
     // Regardless of the event, it will invalidate that particular cache entry.
-    this._cache.del(absPath)
+    this._cache.del(absPath).catch(err => console.error(err))
```

Comment:
This should be a `log` call, not a console log. We'd need to be able to track them in the logging.